errors.md

```diff
@@ -148,11 +148,40 @@ if they are set.
     * The pipeline name is found from the Nextflow config `manifest.homePage`,
       which assumes that the URL is in the format `github.com/nf-core/[pipeline-name]`
     * Example: For `github.com/nf-core/test` the conda environment name should be `nfcore-test`
-* Each dependency listed must have a version number pinned, eg. `toolname=1.6.8`
 
 Each dependency is checked using the [Anaconda API service](https://api.anaconda.org/docs).
-Dependency sublists (eg. `- pip`) are ignored.
-Dependency-specific channels (eg. `conda-forge::openjdk`) are ok.
+Dependency sublists are ignored with the exception of `- pip`: these packages are also checked
+for pinned version numbers and checked using the [PyPI JSON API](https://wiki.python.org/moin/PyPIJSON).
 
-* (Test failure) The package cannot be found on any of the listed channels
+Note that conda dependency with pinned channels (eg. `conda-forge::openjdk`) are fine
+and should be handled by the linting properly.
+
+Each dependency can have the following lint failures and warnings:
+
+* (Test failure) Dependency does not version number pinned, eg. `toolname=1.6.8`
```

Comment:
* (Test failure) Dependency does not version number pinned, eg. `toolname=1.6.8` 

Should be (?)

 * (Test failure) Dependency does not match version number pinned, eg. `toolname=1.6.8`

Reply:
I think I meant "does not _**have a**_ version number pinned"

![27cscr](https://user-images.githubusercontent.com/465550/38080357-84c816e8-3341-11e8-8355-3d04950d56f2.jpg)